src/client/app/components/FilterComponent.jsx

```diff
@@ -21,6 +21,7 @@ class FilterComponent extends React.Component {
 		Gender:&nbsp;
 		<select name="gender" onChange={(e) => _this.handleChange('gender', e.target.value)}>
 			<option value=""></option>
```

Comment:
Let's get rid of the blank values. I don't think they serve a purpose if we have an "All" option.

Reply:
Fair enough. I'll update that.